src/components/pages/MapView.tsx

```diff
@@ -15,7 +16,7 @@ function MapView() {
             return null;
         }
         return (
-            <Marker key={webcam.osmID} position={[webcam.lat, webcam.lon]} icon={MarkerIcon}>
+            <Marker key={webcam.osmID} position={[webcam.lat, webcam.lon]} icon={MarkerIcon} rotationAngle={webcam.direction-90} rotationOrigin={"center center"}>
```

Comment:
I don't like the -90 on the rotation. Partly because it is not explained and partly because it makes the code "ugly."

Although, while typing this comment, I don't have a solution.
turning the SVG results in a required 90deg for cameras that don't have a direction
and baking it into the JSON make it so that you need an understanding of the implementation to use the data

So, most likely, I will be dropping this comment.